.github/workflows/main.yml

```diff
@@ -109,7 +110,18 @@ jobs:
           echo "CC=clang" >> $GITHUB_ENV
           echo "CXX=clang++" >> $GITHUB_ENV
         fi
-
+    
```

Comment:
Minor: Unintended whitespace changes here?

Reply:
This should be okay. Adds a space between steps.

Reply:
The original newline is good, but we don't need the additional 4 spaces as well...  :)